modules/installation-launching-installer.adoc

```diff
@@ -69,7 +69,8 @@ You can run the `create cluster` command of the installation program only once,
 
 .Prerequisites
 
-* Configure an account with the cloud platform that hosts your cluster.
+ifndef::osp[* Configure an account with the cloud platform that hosts your cluster.]
+
```

Comment:
Don't you need to do some credential work on OSP, too? Otherwise, this looks reasonable to me.